src/components/_header.scss

```
@@ -5,84 +5,61 @@
 // Browser support: IE10+, Safari6+, Firefox30+, Chrome30+
 //
 // Markup:
-// <header class="sb-header">
-//   <button class="sb-header__back" aria-label="Back">
-//     <svg class="sb-icon" viewBox="0 0 9 16">
-//       <path d="M8.02631579,16 C8.13407895,16 8.24181579,15.9588947 8.32402632,15.8766579 C8.48847368,15.7122368 8.48847368,15.4456316 8.32402632,15.2812105 L1.04281579,8 L8.32402632,0.718763158 C8.48847368,0.554342105 8.48847368,0.287736842 8.32402632,0.123315789 C8.15957895,-0.0411052632 7.893,-0.0411315789 7.72857895,0.123315789 L0.149631579,7.70226316 C-0.0148157895,7.86668421 -0.0148157895,8.13328947 0.149631579,8.29771053 L7.72857895,15.8766579 C7.81081579,15.9588947 7.91855263,16 8.02631579,16 L8.02631579,16 Z"></path>
-//     </svg>
-//   </button>
-//   <div class="sb-header__content">
-//     <div class="sb-header__text">
-//       <h1 class="sb-header__title sb-title">Header title</h1>
-//       <p class="sb-header__info sb-text">Info regarding this page.</p>
-//     </div>
-//     <button href="#" class="sb-header__btn sb-btn sb-btn--primary">Action button</button>
+// <header class="sb-header sb-border-light-1">
+//   <div class="sb-header__start">
+//    <button class="sb-btn--subtle sb-btn--s" aria-label="Back">
+//      <svg class="sb-icon" viewBox="-4 -1.909 16 16">
+//        <path d="M6.7071 10.3838c.3905.3905.3905 1.0237 0 1.4142-.3905.3905-1.0237.3905-1.4142 0l-5-5c-.3905-.3905-.3905-1.0237 0-1.4142l5-5c.3905-.3905 1.0237-.3905 1.4142 0 .3905.3905.3905 1.0237 0 1.4142L2.4142 6.091l4.293 4.2928z"/>
+//      </svg>
+//    </button>
 //   </div>
+//   <div class="sb-header__center">
+//    <h1 class="sb-header__title sb-title">Header title</h1>
+//   </div>
+//   <div class="sb-header__end">
+//    <button href="#" class="sb-btn sb-btn--primary">Action button</button>
+//   </div
 // </header>
 //
 // Styleguide Components.header
 
 .#{$ns}header {
-  box-shadow: $sb-box-shadow-subtle;
   overflow: auto;
   background: $sb-color-white;
   display: flex;
   z-index: $sb-z-header;
   min-height: sb-px2rems($sb-header-height);
+  align-items: center;
 }
 
-.#{$ns}header__title,
-.#{$ns}header__info {
-  margin-right: sb-rem-spacing('s');
+.#{$ns}header__start {
+  min-width: sb-rem-spacing('xl');
+  padding: sb-rem-spacing('xxs');
+  box-sizing: border-box;
+  flex: 1;
 }
 
-.#{$ns}header__btn {
-  margin-top: sb-rem-spacing('s');
+.#{$ns}header__center {
+  overflow: hidden;
+  flex: 100;
```

```suggestion
  flex: 1 0 0;
```
if you are trying to grow the center element

---

should `flex: 1` do?

---

Yes, `flex: 1` is enough since you really only need to specify `flex-grow`.

---

So I can mark this resolved